lib/src/main/kotlin/com/swmansion/starknet/data/TypedData.kt

```diff
@@ -186,6 +186,7 @@ data class TypedData private constructor(
             Json.decodeFromJsonElement<Revision>(it)
         } ?: Revision.V0
 
+        @Transient
```

Comment:
This has been added to avoid including `separatorName` when serializing `Domain`.